qt/scientific_interfaces/Inelastic/QENSFitting/FitTabConstants.h

```diff
@@ -76,8 +76,8 @@ inline auto templateSubTypes() {
 
 } // namespace IqtFit
```

Comment:
Should this namespace be changed to `Iqt` or similar?

Reply:
I have just left as IqtFit as the model is similar.